src/app/action-item/action-item.component.spec.ts

```diff
@@ -11,50 +11,113 @@ import {VstsPullRequest} from '../../domain/action-item';
 
 describe('action items', () => {
 
-  it('vsts items without tag should not indicate no mergie', () => {
-    const pr: any = { repository: {name: 'test'} };
-    pr.title = 'happy cat code change';
-    const vstsItem = new VstsPullRequest(pr);
-    expect(vstsItem.do_not_merge).toBeFalsy('There should be no indication of merge restricting');
-  });
+  describe('for VstsPullRequest items', () => {
+    it('vsts items without tag should not indicate no mergie', () => {
```

Comment:
so many negatives in this clause. somewhere, shakespeare is smiling proudly

Reply:
hahahahaha